### PR TITLE
DDF-2614: Update asciidoctor dependencies

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Updates the asciidoctor dependencies and fixes windows build issues.
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
@brendan-hofmann @emanns95 @ricklarsen 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl 
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by shaundmorris** [comment]
@AzGoalie for 2.9.x, might have to look around for the properties a bit more. I assume we want to upgrade there as well: https://github.com/codice/ddf/blob/2.9.x/distribution/docs/pom.xml#L59

**Comment by stustison** [comment]
Yes, we'll need to backport this.

**Comment by shaundmorris** [comment]
@AzGoalie looks like asciidoctor 1.5.4 is out now, can you try that one?

File: pom.xml [hunk 1]
```diff
@@ -158,10 +158,9 @@
         <ddf-spatial>DDF Spatial</ddf-spatial>
         <ddf-ui>DDF Search UI</ddf-ui>
 
-        <asciidoctor.maven.plugin.version>1.5.2</asciidoctor.maven.plugin.version>
-        <asciidoctorj-pdf.version>1.5.0-alpha.11</asciidoctorj-pdf.version>
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
-        <jruby.version>9.1.2.0</jruby.version>
+        <asciidoctor.maven.plugin.version>1.5.3</asciidoctor.maven.plugin.version>
+        <asciidoctorj.pdf.version>1.5.0-alpha.11</asciidoctorj.pdf.version>
+        <jruby.version>1.7.25</jruby.version>
```
Comment (AzGoalie) [inline pom.xml]: JRuby's releases are weird, it goes from 1.7.26 straight to 9.1.6.0. Asciidoctor uses 1.7.25 though.

**Pull request closed without merging**